zoos/bears.js

```diff
@@ -0,0 +1,127 @@
+const router = require('express').Router();
+const knex = require('knex'); 
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
```

Comment:
the config object is declared properly, but now that you're having to declare it in both routers, you're not being very DRY -- this can be extracted into a dbConfig file (or name of your choosing) and exported, then imported in each router file to trim them down